mandatory/4-tax.js

```diff
@@ -5,9 +5,8 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {
-  const priceWithTax = 1.2 * price;
-  return priceWithTax;
+function calculateSalesTax(price) {
+    return price + (price / 100 * 20);
```

Comment:
I have done this exercise same as you but  I think its better to use parentheses in a way to make your  math more clear